frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/useInitialAutoLayout.ts

```diff
@@ -29,21 +33,25 @@ export const useInitialAutoLayout = () => {
   const { handleLayout } = useAutoLayout()
 
   useEffect(() => {
-    if (initializeComplete) {
-      return
+    const initialize = async () => {
+      if (initializeComplete) {
+        return
+      }
+
+      const tableNameFromUrl = getActiveTableNameFromUrl()
+      updateActiveTableName(tableNameFromUrl)
+      const hiddenNodeIds = await getHiddenNodeIdsFromUrl()
+      addHiddenNodeIds(hiddenNodeIds)
+
+      const fitViewOptions = tableNameFromUrl
+        ? { maxZoom: 1, duration: 300, nodes: [{ id: tableNameFromUrl }] }
+        : undefined
+
+      if (nodesInitialized) {
+        handleLayout(fitViewOptions, hiddenNodeIds)
+      }
     }
 
-    const tableNameFromUrl = getActiveTableNameFromUrl()
-    updateActiveTableName(tableNameFromUrl)
-    const hiddenNodeIds = getHiddenNodeIdsFromUrl()
-    addHiddenNodeIds(hiddenNodeIds)
-
-    const fitViewOptions = tableNameFromUrl
-      ? { maxZoom: 1, duration: 300, nodes: [{ id: tableNameFromUrl }] }
-      : undefined
-
-    if (nodesInitialized) {
-      handleLayout(fitViewOptions, hiddenNodeIds)
-    }
+    initialize()
```

Comment:
📝 Since async/await is now required, I just wrapped it in a function named initialize.